src/rgw/rgw_perf_counters.cc

```diff
@@ -3,69 +3,212 @@
 
 #include "rgw_perf_counters.h"
 #include "common/perf_counters.h"
+#include "common/perf_counters_key.h"
 #include "common/ceph_context.h"
 
 PerfCounters *perfcounter = NULL;
+ceph::perf_counters::PerfCountersCache *user_counters_cache = NULL;
+ceph::perf_counters::PerfCountersCache *bucket_counters_cache = NULL;
+std::string rgw_op_counters_key = "rgw_op";
```

Comment:
static strings should either be `const std::string` so they can't be changed, or preferably `constexpr std::string_view` so they don't have to be initialized on startup